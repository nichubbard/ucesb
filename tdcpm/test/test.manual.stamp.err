0x55b20cde22a0 7.0000
0x55b20cde7340 1.0000
0x55b20cde7348 2.0000
0x55b20cde23d0 3.0000
0x55b20cde23f0 4.0000
0x55b20cde2408 5.0000
0x55b20cde7358 80.0000
0x55b20cde7378 9.0000
#: 61 lu: 487, h: 599(=1.2x) hm: 539(=0.9x) fm: 426(=0.8x)
