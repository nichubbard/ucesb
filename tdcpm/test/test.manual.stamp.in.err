0x5605cd0a32a0 7.0000
0x5605cd0a8340 1.0000
0x5605cd0a8348 2.0000
0x5605cd0a33d0 3.0000
0x5605cd0a33f0 4.0000
0x5605cd0a3408 5.0000
0x5605cd0a8358 80.0000
0x5605cd0a8378 9.0000
