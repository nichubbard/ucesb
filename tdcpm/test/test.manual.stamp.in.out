r.a = 7;
s(1).k = 1 ns/ch;
s(1).m = 2 ns;
r.d.k = 3 ns/ch;
r.e(2).k = 4 MeV/ch;
r.e(3).m = 5 MeV;
s(2).m = 80 ps;
s(4).m = 9 ps;
r.g = 10;
r.a.a = 10;
r.a(2).a = 10;
r(2).a = 10;
r.e(2).q = 5 MeV/ch;
s.k = 1 ns/ch;
s(8).m = 80 ps;
r.e(5).k = 4 MeV/ch;
TEST(2) = 1 ns;
TEST(3) = 20 ps;
TEST(4) = 0.3 /ps;
POS = 5 cm/ns;
POS = 6.23 dm;
POS = 1.003 ns;
POS = 2004 ps;
POS = 5.002 GeV;
POS = 6.002e+06 keV;
POS = 6 ns;
TEST = 42 cm/ns*ns;
POS = 14 cm/ns;
POS = 48 cm/ns;
POS = 49 cm;
POS = 50 cm/ns;
POS = 51 ns^2;
POS = 46.67 cm/ns;
TEST = 47.67 ns/cm;
POS = 48.67 ns;
POS = 49.67;
POS = 51.5 cm/ns;
POS = 53.67 MeV/ch;
POS = 5.117e+04 mrad;
POS(2) = 7 cm/ns;
POS(2)(3) = 9 cm^3/ns^2;
POS(1)(2) = 8 mm*cm/ns;
POS(1)(2).k = 8 mm*cm/ns/kg;
POS(1)(2).k = 8 mm*cm/ns*kg;
POS(1)(2).k = 8 mm/cm/ns*kg;
POS(1)(2).k = 8 mm/cm*ns/kg;
POS(1)(2).k = 8 mm/cm/ns/kg;
POS(2)(2).m = 8 mm*dm;
POS(2)(2).m = 8 mm*dm @ WR 234;
POS(2)(2).m = 8 mm*dm @ START;
POS(2)(2).q(4) = { 8 cm/ns, 9 m/s };
POS(2)(2).q(4) = { 8 cm/ns @ START, 9 m/s @ END };
POS(1)(2).q(7) = 8 cm/ns;
POS(2) = 
  [ k, m ]

POS(2) = 
  [ k, m ]
  { 1 ns*ns/ch^2, 1 m }

POS(3) = 
  [ k, m ]
  { 3 ns/ch, 4 ns }
  { 5 ns/ch, 6 ns }
  { 7 ns/ch, 8 ns }

POS(4) = 
  [ k, m, x ]
  [[ ns/ch, ns, 10 ps ]]
  (1): { 5, 7, 4 }
  (2): { 6, 8, 5 }

POS(5) = 
  [ k, m.y ]
  [[ ns/ch, 5 ns ]]
  { 9, 10 }

POS(6) = 
  [ k.z, m.y ]
  [[ ns/ch, ns ]]
  { 9, 10 }
  { 9, 10 }

POS(7) = 
  [ k @ START, k @ END ]
  [[ ns/ch, ns/ch ]]
  { 11, 12 }
  { 11, 12 }

POS(8) = 
  [ k.l.m @ START, q, k.l.m @ END ]
  [[ ns/ch, ch, ns/ch ]]
  { 11, 1, 12 }
  { 11, 1, 12 }

POS(2) = {
  qqq(2)(3) = 9 cm/ns;
}
POS(3) = {
  .(2)(3) = 9 cm/ns;
}
POS(4) = {
  qqq(2)(3) = 9 cm/ns;
  qqq(2)(3).q = 9 cm/ns;
}
a(4) = {
  b = 3;
  c(3) = 4;
  d = {
    g = {
      h = 3;
      i(2) = 4;
    }
    e = 3;
    f(3) = 4;
  }
}
b = {
  c(4) = {
    d = 
      [ f, g, h ]
      [[ cm, cm, ns ]]
      { 3, 4, 5 }
      (2): { 4, 4, 5 }
      { 6, 4, 5 }
      (5): { 7, 4, 5 }

  }
}
valid( WR 5, WR 7)
{
  c(5).d(5) = 6 ns;
}
b = {
  valid( WR 3, WR 5)
  {
    c(7).d(7) = 8 ns @ START;
    c(7).d(7) = 8 ns @ WR 4;
    c(7).d(7) = 8 ns @ END;
  }
}
